apps/site/lib/site_web/templates/layout/_footer.html.eex

```diff
@@ -1,111 +1,63 @@
-<footer class="m-footer l-footer">
+<footer class="m-footer">
+  <div class="container m-footer__links">
+    <nav class="m-footer__group" aria-labelledby="footer-customer">
+      <div id="footer-customer" class="m-footer__group-title">Customer Support</div>
+      <%= link "Send Us Feedback", to: customer_support_path(@conn, :index), data: [scroll: true]  %>
+      <%= link "Lost & Found", to: cms_static_page_path(@conn, "/customer-support/lost-and-found"), data: [scroll: true]  %>
+      <%= link to: "https://massachusettsdot.mycusthelp.com/WEBAPP/_rs/supporthome.aspx?lp=3&COID=64D93B66", target: "_blank" do %>
+        Request Public Records <%= content_tag(:span, fa("external-link")) %>
+      <% end %>
+      <%= link "Language Services", to: cms_static_page_path(@conn, "/language-services") %>
+    </nav>
+    <nav class="m-footer__group" aria-labelledby="footer-know-us">
+      <div id="footer-know-us" class="m-footer__group-title">Get to Know Us</div>
+      <%= link "Leadership", to: cms_static_page_path(@conn, "/leadership"), data: [scroll: true] %>
+      <%= link "History", to: cms_static_page_path(@conn, "/history"), data: [scroll: true] %>
+      <%= link "Financials", to: cms_static_page_path(@conn, "/financials"), data: [scroll: true]  %>
+      <%= link "Public Meetings", to: cms_static_page_path(@conn, "/events"), data: [scroll: true]  %>
+      <%= link "Press Releases", to: cms_static_page_path(@conn, "/news"), data: [scroll: true]  %>
+      <%= link "Policies & Civil Rights", to: cms_static_page_path(@conn, "/policies"), data: [scroll: true]  %>
+      <%= link "Safety", to: cms_static_page_path(@conn, "/safety") %>
+      <%= link to: "http://www.mbtabackontrack.com/performance/index.html#/home", target: "_blank" do %>
+        Performance <%= content_tag(:span, fa("external-link")) %>
```

Comment:
Didn't see these icons in the mocks. Can this be removed along with the icon on  Request Public Records?

Reply:
Ah, good catch! Will take those out.